src/main/resources/logback.xml

```diff
@@ -19,15 +19,6 @@
                     <pattern>${ISO8601_DATE_FORMAT}</pattern>
                     <fieldName>created</fieldName>
                 </timestamp>
-                <Pattern>
-                    <Pattern>
-                        {
-                        "trace" : "%X{X-B3-TraceId:-}",
-                        "span" : "%X{X-B3-SpanId:-}",
```

Comment:
did you mean to remove this?

Reply:
Yeah, I believe this was used for zipkin as those fields are always empty in our logs